### DISCUSSION TITLE
Teacher application instantly rejected multiple times (?)

### DISCUSSION THREAD

**Discussion by j-cronk** [community/community#52269] [GitHub Education]
Sorry if this is the wrong place for this, but I am having a ton of trouble trying to apply for educator benefits. Every single time I upload a photo, it says it's manipulated and I automatically get rejected. It doesn't seem like there is any way to appeal this through the application site.

I've been full time faculty at DigiPen since 2018. I'm the Technical Design specialization track owner in the Design department and teach all of the BAGD students how to prototype in Unity and Unreal Engine. I'd love to teach them how to use GitHub so they have an alternative to our on-campus SVN accounts.

[https://www.linkedin.com/in/jocronk/](https://www.linkedin.com/in/jocronk/)

More information about me and my department:
[https://www.digipen.edu/about/our-faculty/design](https://www.digipen.edu/about/our-faculty/design)

**Comment by samus-aran** [comment]
Hi @j-cronk 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We would prefer that you submitted evidence that you didn't feel you needed to manipulate to hide personal information. Are you able to acquire alternative pieces of evidence that would be less revealing but still provide evidence of your faculty status? E.g Faculty ID?

We also ask if you have recently updated your profile info to log out and log back in again before re-applying to ensure the application has the most latest updates from your profile.

Let us know how you get on. 👍
